sympy/vector/tests/test_printing.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 from sympy import Integral, latex, Function
```

Comment:
Leave the coding token there. We have coding tokens in a few files that contain non-ASCII characters.